releasenotes/notes/0.9/improvements-in-DAG-drawer-73283f18ba5ba937.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The DAG drawer was enhance in the context of Jupyter notebooks. The parameter filename is now
+    optional and the function will show the result in the notebook.
+  - |
```

Comment:
This should be under the `features` section. It's not an upgrade note because it's not something a user will have to potentially adjust their existing code because of this change. This is a new feature calling `DAGCircuit.draw()` is something they couldn't do before.